client/docs/config.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Could we move this into a folder `swagger` instead of `client/docs` maybe?

Reply:
this is what other projects are using, but I agree - `docs/swagger` would be probably better place.